Seatbelt/Commands/Windows/DsRegCmd.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
```

Comment:
This file wasn't added to the project file. Maybe you cloned a fresh version of the Seatbelt, copied your changes into it, but then forgot to add the file to the project?